systest/tests/smeshing_test.go

```diff
@@ -381,7 +381,12 @@ func genKeys(tb testing.TB, n int) (pks []ed25519.PrivateKey, pubs []ed25519.Pub
 	return pks, pubs
 }
 
-func prepareVesting(tb testing.TB, keys, start, end, initial, total int) vestingAcc {
+const (
+	start = 8
+	end   = 20
+)
```

Comment:
I feel like instead of making these constant the test above should just use different start and end layers for vesting. Wdyt?